api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -87,12 +88,34 @@ void shouldThrowExceptionWhenMultipleLiveClaimsExistForNino() {
         assertThat(exception.getMessage()).contains(existingIds.toString());
     }
 
+    @Test
+    void shouldReturnIneligibleWhenEligibilityServiceReportsIneligible() {
+        EligibilityResponse eligibilityResponse = anEligibilityResponseWithStatus(INELIGIBLE);
+        PaymentCycleVoucherEntitlement voucherEntitlement = aPaymentCycleVoucherEntitlementWithZeroVouchers();
+        given(client.checkEligibility(any())).willReturn(eligibilityResponse);
+        given(duplicateClaimChecker.liveClaimExistsForHousehold(any())).willReturn(false);
+        given(claimRepository.findLiveClaimsWithNino(any())).willReturn(emptyList());
+        given(paymentCycleEntitlementCalculator.calculateEntitlement(any(), any(), any())).willReturn(voucherEntitlement);
+        Claimant claimant = aValidClaimant();
+
+        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateClaimant(claimant);
+
+        assertCorrectResult(result, INELIGIBLE, NOT_CONFIRMED, eligibilityResponse, voucherEntitlement);
+        verify(claimRepository).findLiveClaimsWithNino(claimant.getNino());
+        verify(client).checkEligibility(claimant);
+        verify(duplicateClaimChecker).liveClaimExistsForHousehold(eligibilityResponse);
+        verify(paymentCycleEntitlementCalculator).calculateEntitlement(
+                Optional.ofNullable(claimant.getExpectedDeliveryDate()),
+                eligibilityResponse.getDateOfBirthOfChildren(),
+                LocalDate.now());
+    }
+
     @Test
     void shouldReturnEligibleWhenNotDuplicateAndEligibleWithVouchers() {
         EligibilityResponse eligibilityResponse = anEligibilityResponseWithStatus(ELIGIBLE);
         PaymentCycleVoucherEntitlement voucherEntitlement = aPaymentCycleVoucherEntitlementWithVouchers();
         given(client.checkEligibility(any())).willReturn(eligibilityResponse);
-        given(duplicateClaimChecker.checkForDuplicateClaimsFromHousehold(any())).willReturn(ELIGIBLE);
+        given(duplicateClaimChecker.liveClaimExistsForHousehold(any())).willReturn(false);
```

Comment:
I think there is another missing test here, I don’t see any scenarios where `duplicateClaimChecker.liveClaimExistsForHousehold` returns `true`. Looks like there should be a test where the final `EligibiltyStatus` should be `DUPLICATE` when calling `evaluateClaimant` on `EligibilityAndEntitlementService`.

Reply:
Good spot - we actually had a small bug in the code. Have refactored now.